### ISSUE TITLE
Searching from MpDroid

### ISSUE THREAD

**Issue by Tigger2014** [woutervanwijk/Pi-MusicBox#194]
When i do a search from MpDroid i get no results found but its working the web interface.
if i can provide any logs let me know

Thanks

**Comment by laclaro** [comment]
Hi,

maybe this is the wrong place to ask, but I didn't know any better: Did I get it right, that pimusicbox provides a possibility to search spotify and other services by using an ordinary mpd-client (e.g. MPDroid)? Does it some wrapping around the MPD-protocol?

Best,
Henning

**Comment by kingosticks** [comment]
Yes. It uses Mopidy which [implements most of the MPD protocol](https://docs.mopidy.com/en/latest/ext/mpd/). If you do have any further questions on how to use anything then please make a new post at https://discuss.mopidy.com/c/pi-musicbox, thanks.
